rtc_base/physical_socket_server.cc

```diff
@@ -1711,7 +1711,7 @@ bool PhysicalSocketServer::Wait(webrtc::TimeDelta max_wait_duration,
               }
               if ((wsaEvents.lNetworkEvents & FD_CONNECT) &&
                   wsaEvents.iErrorCode[FD_CONNECT_BIT] != 0) {
-                RTC_LOG(WARNING)
+                RTC_LOG(LS_INFO)
```

Comment:
Not `LS_WARNING`?

Reply:
You're right. I mixed this up with other logging changes we made in this file.